### PR TITLE
Code cleanup - guard headers, CAN printing

### PR DESCRIPTION
**Pull request by brunofalmeida**
- Guard headers for header files prevent errors/warnings from redefining constants and redeclaring functions
- Commented out debugging print statements for CAN because they clutter a lot of the UART output when using the CAN library

### CODE REVIEW

**Comment by siddMahen** [comment]
I think the CAN print statements can be safely removed.

File: include/queue/queue.h [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+#define QUEUE_H
+
 /*
     AUTHORS: J. W. Sheridan, Siddharth Mahendraker, Shimi Smith
 */
 
 #include <stdint.h>
 
-#define MAX_QUEUE_SIZE 10
+#define MAX_QUEUE_SIZE 3
```
Comment (siddMahen) [inline include/queue/queue.h]: Was this change intentional? Making the MAX_QUEUE_SIZE 3 could be problematic down the road, when we realize we need more room.

**Comment by brunofalmeida** [comment]
I think that's fine as long as we don't use more than a couple of queues per repository. I think we had memory space problems while rushing to test the qual demo which had 5 queues in OBC. If we want to have more smaller queues, maybe change the size to 5?

**Comment by siddMahen** [comment]
Ok sounds good
